lib/OpenQA/Setup.pm

```diff
@@ -68,6 +68,7 @@ sub read_config ($app) {
             do_push => 'no',
             do_cleanup => 'no',
             git_auto_clone => 'yes',
+            git_auto_clone_all => 'no',
```

Comment:
Maybe it would make sense to emphasize on the fact that this will *update* existing checkouts (and actually not clone new ones):
```suggestion
            git_auto_update => 'no',
```

I would avoid `_all`. I know what you're getting at here - with this setting more jobs are affected than with just `git_auto_clone` - but to a user `_all` is probably not saying much.

(I think we'll eventually need to describe under what conditions which settings apply and what Git commands they'll execute in a comment in the ini file - but this can be done in a follow-up.)

Reply:
I changed it to `git_auto_update`